partition/roles/sonic/tasks/generate_config_db.yaml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Create temporary directory
+  file:
+    path: /tmp/sonic
+    state: directory
+
+- name: Render input values template
+  template:
+    src: sonic-config.yaml.j2
+    dest: /tmp/sonic/sonic-config.yaml
+
+- name: Copy current config_db.json into temporary directory
+  copy:
+    src: /etc/sonic/config_db.json
+    dest: /tmp/sonic/config_db.json
+    mode: 0644
+    remote_src: true
+
+- name: Run sonic-configdb-utils
+  community.docker.docker_container:
+    name: sonic-configdb-utils
+    image: "{{ sonic_configdb_utils_image_name }}:{{ sonic_configdb_utils_image_tag }}"
```

Comment:
Why running the `sonic-configdb-utils` on the host instead of running it in the `metal-stack-deployment` container?

Reply:
It needs to read the `/etc/sonic` and `/usr/share/sonic/device` directories. The required files could be copied to the control node so `sonic-confidb-utils` could be run there, but what for? What is the benefit of running it inside the `metal-stack-deployment` container?